src/sema/expression/function_call.rs

```diff
@@ -547,20 +547,19 @@ pub(super) fn function_call_named_args(
     Err(())
 }
 
-/// Resolve a method call with positional arguments
-pub(super) fn method_call_pos_args(
+/// Check if the function is a method of a variable
+fn try_variable(
```

Comment:
This is not a good name. We are handling the case of `foo.bar(..)` here, e.g. `super.func()`, `library.libfunc()`, `abi.encode()`. How about `fn try_namespaced()`? It's not great either but a little better.